lib/swagger/info.rb

```diff
@@ -0,0 +1,35 @@
+class Swagger::Info
+  attr_accessor :fields
+
+  def initialize(fields)
+    @fields = Hashie::Mash.new(fields)
+  end
+
+  delegate :description,
```

Comment:
thoughts on making these methods rather than delegations? I find that to be more greppable. (as you might have noticed, Kane and I removed all delegations from our view models during refactor 😄 )